### PR TITLE
PPTP-2391: Fixed threshold date error links

### PR DESCRIPTION
**Pull request by davidmthornton**
1. Fixed error summary links to take focus to  the date entry
2. Stopped empty links from being in the error summary
3. Fixed a type on a message

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -482,7 +482,7 @@ liability.expectToExceedThreshold.two.required.fields = Date must include the {0
 liability.expectToExceedThreshold.one.field = Date must include the {0}
 
 liability.exceededThresholdWeightDate.sectionHeader = Liability details
-liability.exceededThresholdWeightDate.title = When did you you meet the 10,000kg threshold?
+liability.exceededThresholdWeightDate.title = When did you meet the 10,000kg threshold?
```
Comment (liampan) [inline conf/messages.en:485]: is this correct in welsh?

File: app/uk/gov/hmrc/plasticpackagingtax/registration/views/components/errorSummary.scala.html [hunk 2]
```diff
@@ -22,11 +22,15 @@
 
 @errorList = @{errors.map { error =>
     val errorKey = error.key.replace(".", "_").replace('[', '_').replace("]", "")
```
Comment (liampan) [inline app/uk/gov/hmrc/plasticpackagingtax/registration/views/components/errorSummary.scala.html:24]: what is this :S

Comment (liampan) [reply app/uk/gov/hmrc/plasticpackagingtax/registration/views/components/errorSummary.scala.html:24]: this should be the ID of the input field 🤔

File: conf/messages.en (hunk 1 above)
Comment (davidmthornton) [reply conf/messages.en:485]: Looks like it as there aren't any repeated words... But I'm clearly not Welsh.

Pryd gwnaethoch gyrraedd y trothwy 10,000kg?

Comment (liampan) [reply conf/messages.en:485]: Screenshot 2022-07-14 at 13 46 03
Aye looks good

File: app/uk/gov/hmrc/plasticpackagingtax/registration/views/components/errorSummary.scala.html (hunk 2 above)
Comment (liampan) [reply app/uk/gov/hmrc/plasticpackagingtax/registration/views/components/errorSummary.scala.html:24]: if we just removed this replacing logic does it work naturally?

Comment (davidmthornton) [reply app/uk/gov/hmrc/plasticpackagingtax/registration/views/components/errorSummary.scala.html:24]: it does for some of the error links on date but not all - the fix above sends all error links for that field to the date day input

**Review by PSouthern — APPROVED** [review]

**Pull request merged**